src/state.rs

```diff
@@ -31,8 +32,17 @@ pub struct State {
 
 // Where the program state is persisted on disk
 fn path() -> Option<PathBuf> {
+    let mut base_path = dirs::data_local_dir();
+
+    // Overwrite the directory on Windows because it's empty in nanoserver
```

Comment:
Please follow the [style guide](https://github.com/stepchowfun/docuum/blob/main/CONTRIBUTING.md#rust-style-guide) and punctuate sentences—thanks!